src/app/app-modules/core/components/app-footer/app-footer.component.ts

```diff
@@ -70,8 +76,24 @@ export class AppFooterComponent implements OnInit, DoCheck {
           action: 'Save',
         },
       });
-    snackBarRef.afterDismissed().subscribe(() => {
-      //console.log('locsl', JSON.parse(localStorage.getItem('swymedLogin')));
-    });
+    snackBarRef.afterDismissed().subscribe(() => {});
+  }
+
+  getUIVersionAndCommitDetails() {
+    const commitDetailsPath: any = 'assets/git-version.json';
+    this.auth.getUIVersionAndCommitDetails(commitDetailsPath).subscribe(
+      (res: { version?: string }) => {
+        if (res && res.version) {
+          this.commitDetailsUI = res;
```

Comment:
do we need this.commitDeatilsUI which stores the whole response? are using it anywhere